completions/quick-lint-js.ps1

```diff
@@ -0,0 +1,16 @@
+Register-ArgumentCompleter -CommandName quick-lint-js -Native -ScriptBlock {
```

Comment:
You should list this file in `completions/CMakeLists.txt` so it gets added to the Windows .zip downloadable.

Reply:
Is this correct? I haven't worked with CMake.